README.md

```diff
@@ -10,7 +10,11 @@ Build
 
 In order to build EVA, you need to install the Java Development Kit 7 and Maven.
 
-The main dependency is OpenCGA 0.5.2, which you can get from https://github.com/opencb/opencga. Please follow the download/compilation instructions there.
+Our dependencies are OpenCGA and Variation-Commons 
+
+You can get OpenCGA 0.5.2 from https://github.com/opencb/opencga. Please follow the download/compilation instructions there.
```

Comment:
I know it wasn't there before, but could you please specify the branch? People unfamiliar with the EVA - OpenCGA interaction tend to clone the `develop` branch.